js/script.js

```diff
@@ -0,0 +1,2 @@
+const average = (...args) => console.log(args.reduce((x,y) => x + y) / args.length);
```

Comment:
remove this console from here

what is the second param in reduce function? use it ;)